requirements.txt

```diff
@@ -0,0 +1,5 @@
+NEURON
+matplotlib
+numpy
+scipy
+hnn-core
```

Comment:
currently installs v0.1 but we'll soon have v0.2 :)

Reply:
Is `requirements.txt` a binder feature that gets run prior to instantiating the jupyter notebook?